config/config.go

```diff
@@ -72,7 +72,7 @@ type SamplerConfig struct {
 	Param float64 `yaml:"param"`
 
 	// SamplingServerURL is the address of jaeger-agent's HTTP sampling server
-	// Can be set by exporting an environment variable named JAEGER_SAMPLER_MANAGER_HOST_PORT
+	// Can be set by exporting an environment variable named JAEGER_CONFIG_MANAGER_HOST_PORT
 	SamplingServerURL string `yaml:"samplingServerURL"`
```

Comment:
I think this property should be changed as well. Similarly, this should be backwards compatible.

cc @yurishkuro